src/compiler/emitter.ts

```diff
@@ -5153,7 +5181,6 @@ var __param = (this && this.__param) || function (paramIndex, decorator) {
                     }
 
                     if (node.kind === SyntaxKind.ClassDeclaration) {
-                        // TODO: rename block scoped classes

```

Comment:
Comment is remove since it is not relevant - only top level classes should be hoisted and they are not renamed